cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -279,6 +279,7 @@ data:
 {{- else }}
             return 404;
 {{- end }}
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
```

Comment:
Good idea. SAML/OIDC are served by the Aggregator, so we shouldn't include these Nginx routes if Aggregator is disabled. Could you add the same for SAML too? Could we also add some documentation around this requirement?